.github/workflows/cbbr_build.yml

```diff
@@ -45,7 +45,6 @@ jobs:
           python-version: "3.11.1"
 
       - name: install dependencies ...
-        working-directory: db-cbbr/cbbr_build
```

Comment:
Not sure how this ended up in this PR, seems like a git mistake by me but this is an unnecessary line so 🤷

Reply:
this may be required since one of the lines is `pip3 install -r requirements.txt`